MEMBER_REPRESENTATION.md

```diff
@@ -1,27 +1,59 @@
-# Member Representation
+# OpenJS Foundation Representatives
 
-The following document lists OpenJS Foundation members and their representation to any standards organizations:
+The following document lists OpenJS Foundation's memberships and liaison relationships, along with the community members designated to participate on behalf of the Foundation:
+
+* [Ecma International]
+  * [TC39] Delegates
+    * Guy Bedford
+    * Jordan Harband
```

Comment:
"delegates" isn't the right word to use here - the other three are delegates of OpenJS, but i'm an invited expert who also happens to be in the standards group.

Reply:
Maybe I'll remove "Delegates" in these cases then.